quickstart/index.md

```diff
@@ -74,13 +74,13 @@ The API is easy to use. From a CoreOS machine, you can simply use curl to set an
 Set a key `message` with value `Hello world`:
 
 ```sh
-curl -L http://127.0.0.1:4001/v2/keys/message -XPUT -d value="Hello world"
+curl -L http://127.0.0.1:2379/v2/keys/message -XPUT -d value="Hello world"
```

Comment:
Hmm, I wonder why these are curl to begin with, instead of etcdctl. I wonder if we should make one of them etcdctl and keep one curl to show that you don't have to use your tool.

In order to do that we'd probably need to add a sentence about what etcdctl is and then that you can also use any regular http stuff like curl.

"etcdctl is a command line interface to etcd that is preinstalled on coroes. to set a key, blah blah:"